sdk/Lusid.Drive.Sdk/Extensions/FilesApiExtensions.cs

```diff
@@ -0,0 +1,263 @@
+            string xLusidDrivePath,
+            int? contentLength,
+            Stream body)
+        {
+            // Verify that 'xLusidDriveFilename' is set
+            if (xLusidDriveFilename == null)
+            {
+                throw new ApiException(400, 
+                    "Missing required parameter 'xLusidDriveFilename' when calling ExtensionsMethods->CreateFile");
+            }
+
+            // Verify that 'xLusidDrivePath' is set
+            if (xLusidDrivePath == null)
+            {
+                throw new ApiException(400,
+                    "Missing required parameter 'xLusidDrivePath' when calling ExtensionsMethods->CreateFile");
+            }
+
+            // Verify that 'contentLength' is set
+            if (contentLength == null)
+            {
+                throw new ApiException(400, 
+                    "Missing required parameter 'contentLength' when calling ExtensionsMethods->CreateFile");
+            }
+
+            // Verify that 'body' is set
+            if (body == null)
+            {
+                throw new ApiException(400, 
+                    "Missing required parameter 'body' when calling ExtensionsMethods->CreateFile");
+            }
+
+            // Endpoint to POST new file
+            const string localVarPath = "./api/files";
+            
+            // Set default parameters
+            var localVarHeaderParams = new Dictionary<string, string>(api.Configuration.DefaultHeader);
+
+            // Set the Content-Type header
+            var localVarHttpContentTypes = new[] {"application/octet-stream"};
+            var localVarHttpContentType = api.Configuration.ApiClient.SelectHeaderContentType(localVarHttpContentTypes);
+
+            // Set the Accept header
+            var localVarHttpHeaderAccepts = new[] {"text/plain", "application/json", "text/json"};
+            var localVarHttpHeaderAccept = api.Configuration.ApiClient.SelectHeaderAccept(localVarHttpHeaderAccepts);
+            
+            if (localVarHttpHeaderAccept != null)
+            {
+                localVarHeaderParams.Add("Accept", localVarHttpHeaderAccept);
+            }
+            
+            localVarHeaderParams.Add("x-lusid-drive-filename", api.Configuration.ApiClient.ParameterToString(xLusidDriveFilename));
+            localVarHeaderParams.Add("x-lusid-drive-path", api.Configuration.ApiClient.ParameterToString(xLusidDrivePath));
+            localVarHeaderParams.Add("Content-Length", api.Configuration.ApiClient.ParameterToString(contentLength));
+            
+            // Authentication (oauth2) required
+            if (!string.IsNullOrEmpty(api.Configuration.AccessToken))
+            {
+                localVarHeaderParams["Authorization"] = "Bearer " + api.Configuration.AccessToken;
+            }
+
+            // Set the LUSID header
+            localVarHeaderParams["X-LUSID-Sdk-Language"] = "C#";
+            localVarHeaderParams["X-LUSID-Sdk-Version"] = "0.11.2617";
```

Comment:
this looks like its hardcoded which is incorrect, it should reflect the latest version of the SDK/api